src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -484,6 +480,45 @@ def _restart_qemu(self):
     start_qemu()
     self._setup_device_and_fuzzer()
 
+  def _corpus_target_subdir(self, relpath):
+    return os.path.join(self.corpus_directories_target, relpath)
+
+  def _init_corpus_info(self, corpus_directories):
+    """ Initialize corpus info for fuzzing and merging. """
+    self.corpus_directories = corpus_directories
```

Comment:
we shouldn't be storing `corpus_directories` here, since the Runner class should not be tied to any particular invocation. We can just pass this as arguments to everything that needs it.

Reply:
got it, made individual functions to calculate these vars